misk/src/test/kotlin/misk/web/JsonForProtoEndpointsTest.kt

```diff
@@ -79,14 +129,64 @@ internal class JsonForProtoEndpointsTest {
     }
   }
 
+  @Test
+  fun `grpc to grpc endpoint`() {
+    val requestBody = Shipment.Builder()
+        .shipment_token("abc")
+        .build()
+    val expectedResponseBody = Warehouse.Builder()
+        .warehouse_token("abc")
+        .build()
+
+    val grpcClient = GrpcClient.Builder()
+        .baseUrl(jettyService.httpsServerUrl!!)
+        .client(httpClient)
+        .build()
+    val shippingClient = grpcClient.create(ShippingClient::class)
+
+    val responseBody = shippingClient.GetDestinationWarehouse().executeBlocking(requestBody)
+    assertThat(responseBody).isEqualTo(expectedResponseBody)
+  }
+
   class TestModule : KAbstractModule() {
     override fun configure() {
-      install(WebTestingModule())
-      install(WebActionModule.create<EchoShipmentToken>())
+      install(WebTestingModule(webConfig = WebTestingModule.TESTING_WEB_CONFIG.copy(
+          http2 = true
+      )))
+      install(WebActionModule.create<ProtoEchoShipmentToken>())
+      install(WebActionModule.create<GrpcEchoShipmentToken>())
     }
   }
 
-  class EchoShipmentToken @Inject constructor() : WebAction {
+  // NB: The server doesn't get a port until after it starts so we create the client module
+  // _after_ we start the services
+  class ClientModule(val jetty: JettyService) : KAbstractModule() {
```

Comment:
does this class still need to be here? can't see where it used..

Reply:
ooooh good catch. Fixed!